ansible/roles/operator-pipeline/templates/openshift/tasks/decide-index-paths.yml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: decide-index-paths
+spec:
+  params:
+    - name: pipeline_image
+  results:
+    - name: index_image_paths
+      description: Comma-separated list of index reference + temporary location of the unpublished images generated by IIB builds
+
+  workspaces:
+    - name: input
+  steps:
+    - name: decide-paths
+      image: "$(params.pipeline_image)"
+      workingDir: $(workspaces.input.path)
+      script: |
+        #! /usr/bin/env bash
+        set -xe
+
+        echo "Checking which build actually ran, FBC change or operator/bundle change."
+
+        LEGACY_BUILD="$(workspaces.input.path)/index-image-paths.txt"
```

Comment:
Don't call it legacy. It is one of the supported formats and calling it a legacy may confuse users.

Reply:
Changed